src/gui/element/settings.rs

```diff
@@ -30,14 +31,26 @@ pub fn data_container<'a, 'b>(
     self_update_channel_state: &'a mut SelfUpdateChannelState,
     default_addon_release_channel_picklist_state: &'a mut pick_list::State<GlobalReleaseChannel>,
     reset_columns_button_state: &'a mut button::State,
+    localization_state: &'a mut LocalizationState,
 ) -> Container<'a, Message> {
     let mut scrollable = Scrollable::new(scrollable_state)
         .spacing(1)
         .height(Length::FillPortion(1))
         .style(style::Scrollable(color_palette));
 
+    let ctx = static_json_gettext_build!(
+        "en_US",
+        "en_US",
+        "locale/en_US.json",
+        "da_DK",
+        "locale/da_DK.json"
+    )
+    .unwrap();
+    let val = localization_state.languages.get(&config.language).unwrap();
+    let a = get_text!(ctx, val, "hello").unwrap();
```

Comment:
This is the core of the way we will use this system, @tarkah.
It works simply and nice, imho.

However I need to find a proper place to generate that `ctx` which I need to pass into `get_text!` macro everywhere I use it. Any idea of something nice?

Reply:
Super simple! This is awesome. Can we store the JSONGetText (ctx) struct in Ajour? We can build it / store it during  Application::new ?